authority-portal-keycloak/sovity-theme/login/register.ftl

```diff
@@ -0,0 +1,187 @@
+<#import "template.ftl" as layout>
+<#import "register-commons.ftl" as registerCommons>
+<@layout.registrationLayout displayMessage=!messagesPerField.existsError('firstName','lastName','email','username','password','password-confirm','termsAccepted'); section>
+    <#if section = "header">
+       
+    <#elseif section = "form">
+        <div class="formContainer">
+            <div class="formHeader">
+                    <h3>REGISTRATION FORM</h3>
+            </div>
+            <div class="formBody">
+                    <div id="kc-form">
+                        <form id="kc-register-form" action="${url.registrationAction}" class="formContainer" method="post">
+                        <#-- First name -->
+                        <div class="${properties.kcFormGroupClass!}">
+                            <div class="${properties.kcInputWrapperClass!}">
+                                <input type="text" id="first-name" placeholder="First Name"
+                                       class="text-input"
+                                       name="firstName"
+                                       value="${(register.formData.firstName!'')}"
+                                />
+                            </div>
+                        </div>
+                        <span id="fe-input-error-first-name" class="${properties.kcInputErrorMessageClass!}">
+                            Please specify first name.
+                        </span>
+
+                        <#-- Last name -->
+                        <div class="${properties.kcFormGroupClass!}">
+                            <div class="${properties.kcInputWrapperClass!}">
+                                <input type="text" id="last-name" placeholder="Last Name" class="text-input"
+                                       name="lastName"
+                                       value="${(register.formData.lastName!'')}"
+                                />
+                            </div>
+                        </div>
+                        <span id="fe-input-error-last-name" class="${properties.kcInputErrorMessageClass!}">
+                            Please specify last name.
+                        </span>
```

Comment:
Please provide **a** first name/last name

Reply:
This page is actually no longer used iirc but updating it just in case